flexmeasures/app.py

```diff
@@ -97,6 +100,17 @@ def create(  # noqa C901
     if app.env not in ("documentation", "development"):
         SSLify(app)
 
+    # Prepare profiling, if needed
+
+    if app.config.get("FLEXMEASURES_PROFILE_REQUESTS", False):
+        Path("profile_reports").mkdir(parents=True, exist_ok=True)
```

Comment:
Where would this be if I turn on FLEXMEASURES_PROFILE_REQUESTS on a server where flexmeasures runs as pip-installed package? Probably Flask's instance directory. But I'm just guessing.

In any case, we can add a note where to expect this folder to the configuration docs.